include/caf/led_effect.h

```diff
@@ -95,7 +129,7 @@ struct led_effect {
 	{									\
 		.steps = ((const struct led_effect_step[]) {			\
 			{							\
-				.color = _color,				\
+				.color = LED_COLOR_USE(_color),			\
```

Comment:
will effects defined by others have to use the new macro?

Reply:
Totally rebuilt right now.